drivers/temperature/adt7420/adt7420.c

```diff
@@ -3,7 +3,7 @@
  *   @brief  Implementation of ADT7420 Driver.
  *   @author DBogdan (dragos.bogdan@analog.com)
 ********************************************************************************
- * Copyright 2012(c) Analog Devices, Inc.
```

Comment:
Change the commit title to: `drivers:temperature:adt7420:Add new supported devices` or something similar but with the path for the modifications and the current commit message can go into the description with the lines wrapped to 75 characters.

Reply:
Modified the commit message as suggested